website/docs/d/oracle_database_system_versions.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "Oracle"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_oracle_database_system_versions"
+description: |-
+  Gets information about existing Oracle Database Systems Versions.
+---
+
+# Data Source: azurerm_oracle_database_system_versions
+
+Use this data source to access information about existing Oracle Database Systems Versions.
+
+## Example Usage
+
+```hcl
+data "azurerm_oracle_db_system_versions" "example" {
+  location = "eastus"
+}
+
+output "id" {
+  value = data.azurerm_oracle_db_system_versions.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `location` - (Required) The Azure Region to query for the Oracle Database Systems Versions.
+
+---
+
+* `database_software_image_support_enabled` - (Optional) If true, filters the results to the set of Oracle Database versions that are supported for the database software images.
+
+* `database_system_id` - (Optional) The DB system AzureId. If provided, filters the results to the set of database versions which are supported for the DB system.
+
+* `database_system_shape` - (Optional) If provided, filters the results to the set of database versions which are supported for the given shape. e.g., VM.Standard.x86.
+
+* `shape_family` - (Optional) If provided, filters the results to the set of database versions which are supported for the given shape family.
+
+* `storage_management` - (Optional) The DB system storage management option. Used to list database versions available for that storage manager. Valid values are ASM and LVM.
```

Comment:
The description should be `Possible values are ...` ([ref](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-documentation-standards.md#descriptions))